CHANGELOG.md

```diff
@@ -6,7 +6,8 @@
 - Add `flowlog` to `ionoscloud_nic` resource
 - Add `flowlog` to `ionoscloud_networkloadbalancer` resource
 - Add `flowlog` to `ionoscloud_application_loadbalancer` resource
-- 
+- Add parameters for private k8s cluster: `public`, `location`, `nat_gateway_ip`, `node_subnet`
```

Comment:
it will be part of the 6.4.12 release now

Reply:
@cbeti-ionos done in 915ce12afd765de96f4189af2427dccf516874d1